Sources/Basics/HTTPClient/URLSessionHTTPClient.swift

```diff
@@ -0,0 +1,379 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
```

Comment:
This file was renamed from `HTTPClient+URLSession.swift`, but GitHub didn't pick up the renaming and displays it as a new file for some reason.